src/configure-balena-host-envvars.sh

```diff
@@ -58,6 +59,7 @@ HOST_VALUES=(
     db
```

Comment:
Why are there a bunch of duplicate entries in this array?

Reply:
We've several envvars that point to the same host, so those that have different envvar names need the same value.

Reply:
(For example `BALENA_DELTA_S3_HOST` and `BALENA_IMAGE_MAKER_S3_HOST`). Ideally of course, these would be one envvar, but this needs auditing to make sure all those components *can* use the same envvar...

Reply:
Have you thought about making this clearer with
```bash
declare -A HOST_ENVVARS
HOST_ENVVARS[BALENA_API_HOST]=api
HOST_ENVVARS[BALENA_UI_HOST]=dashboard
...

for VARNAME in "${!HOST_ENVVARS[@]}"; do
   VARVALUE=${HOST_ENVVARS[$VARNAME]}
   ...
done

Reply:
Yeah, that's not a bad idea. It kind of grew organically, but that's a better way of doing it.

Reply:
Done.